smartsim/_core/_cli/build.py

```diff
@@ -298,3 +310,26 @@ def check_tf_install(self):
                 )
         except SetupError as e:
             logger.warning(str(e))
+
+    def check_backends_install(self):
+        """Checks if backends have already been installed.
+        Logs details on how to proceed forward
+        if the RAI_PATH environment variable is set or if
+        backends have already been installed.
+        """
+        if os.environ.get("RAI_PATH"):
+            if installed_redisai_backends():
+                logger.error(
+                    f"Backends are already built and loaded at '{CONFIG.redisai}'. There is no need to build."
```

Comment:
mention that they are specified in the environment at ``RAI_PATH`` so the user is aware thats where they are specified.